### PR TITLE
GOV-287 - Added kong-admin postman collection

### PR DESCRIPTION
**Pull request by danishjamal104**
## Description

* Added kong-admin postman collection

This PR is sidecar for the actual story in [JIRA-GOV-287](https://mifosforge.jira.com/browse/GOV-287)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Design related bullet points or design document link related to this PR added in the description above.

- [x] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [x] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by danishjamal104** [comment]
@fynmanoj please review and merge. Thanks :)

File: PostmanCollections/External/Kong CE Admin API V2.0.postman_collection.json
Comment (fynmanoj) [inline PostmanCollections/External/Kong CE Admin API V2.0.postman_collection.json:1]: We had an issue in the latest version of postman collection that it is giving error while importing to POSTMAN. can you check whether this issue is resolved. there is a chance your changes are infected when you did rebase

**Review by fynmanoj — CHANGES_REQUESTED** [review]
clarification requested

File: PostmanCollections/External/Kong CE Admin API V2.0.postman_collection.json
Comment (danishjamal104) [reply PostmanCollections/External/Kong CE Admin API V2.0.postman_collection.json:1]: checking

Comment (danishjamal104) [reply PostmanCollections/External/Kong CE Admin API V2.0.postman_collection.json:1]: @fynmanoj its importing. Working fine for me.

**Pull request merged**